atom/browser/resources/win/atom.manifest

```diff
@@ -32,7 +32,6 @@
 
   <asmv3:application xmlns:asmv3="urn:schemas-microsoft-com:asm.v3">
     <asmv3:windowsSettings xmlns="http://schemas.microsoft.com/SMI/2005/WindowsSettings">
-      <dpiAware>true</dpiAware>
```

Comment:
> you can use <dpiAware>True/PM</dpiAware> instead of adding back the code

I didn't know this, it is much better!

> PerMonitor tag + a call to EnableNonClinetDpiScaling() on 1607
> PerMonitorV2 on 1703 (no need for that additional API call)

I agree it is important to have these, but I'm not sure whether they work well with Chromium's own UI framework. Can someone who has Windows running on multiple monitors test it?